### PR TITLE
Capture CSP billing adapter config

### PR DESCRIPTION
**Pull request by jgleissner**
Is this everything we need?

### CODE REVIEW

File: supportconfig/plugins/suse_public_cloud [hunk 1]
```diff
@@ -301,6 +301,18 @@ function update_infra_data() {
     zypper ref >> $INFO_FILE
 }
 
+function csp_billing_adapater() {
```
Comment (smarlowucf) [inline supportconfig/plugins/suse_public_cloud:304]: s/adapater/adapter

**Comment by smarlowucf** [comment]
> Is this everything we need?

I think so

File: supportconfig/plugins/suse_public_cloud [hunk 2]
```diff
@@ -301,6 +301,18 @@ function update_infra_data() {
...
+function csp_billing_adapter() {
+    local INFO_FILE="$DATA_LOCATION/csp_billing_adapter_config.txt"
+    local BA_CONFIG_FILE="/etc/csp_billing_adapter/config.yaml"
+    echo "#==[ CSP billing adapter config ]========================#" > $INFO_FILE
+    if [ -e $BA_CONFIG_FILE ]; then
+        echo "# cat $BA_CONFIG_FILE" >> $INFO_FILE
+        cat $BA_CONFIG_FILE >> $INFO_FILE
+    else
+        echo "# not found" >> $INFO_FILE
+    fi
```
Comment (rjschwei) [inline supportconfig/plugins/suse_public_cloud:313]: We also want the log file the billing adapter creates.

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: supportconfig/plugins/suse_public_cloud (hunk 2 above)
Comment (jgleissner) [reply supportconfig/plugins/suse_public_cloud:313]: What file would that be?

Comment (rjschwei) [reply supportconfig/plugins/suse_public_cloud:313]: Where ever @jesusbv who is implementing the -local csp adapter part decides to write the logging information. Somewhere in `/var/log`. `cspadapate.log` would make sense to me but I have not checked up on the state of the `-local` implementation for the adapter.

Comment (smarlowucf) [reply supportconfig/plugins/suse_public_cloud:313]: File based logging is not implemented yet. Based on other naming choices I would suggest something along the lines of `/var/log/csp_billing_adapter.log` or `/var/log/csp-billing-adapter.log`

Comment (rjschwei) [reply supportconfig/plugins/suse_public_cloud:313]: @jgleissner pick one either '_' or '-' then it is prescribed for the local logging implementation, whomever does it has to use the name picked here.

**Review by rjschwei — APPROVED** [review]

**Review by smarlowucf — APPROVED** [review]

**Pull request merged**